lib/capybara/webkit/driver.rb

```diff
@@ -359,7 +359,11 @@ def modal_action_options_for_browser(options)
     end
 
     def default_wait_time
-      Capybara.respond_to?(:default_max_wait_time) ? Capybara.default_max_wait_time : Capybara.default_wait_time
+      if respond_to?(:session_options) && session_options
+        session_options.default_max_wait_time
+      else
+        Capybara.respond_to?(:default_max_wait_time) ? Capybara.default_max_wait_time : Capybara.default_wait_time
```

Comment:
@mhoran The default_wait_time deprecation happened in 2.5.0  - If you want to move to supporting only capybara >= 2.5.0 (August 2015) - then it could be simplified